docs/search/query-design/boosting.md

```diff
@@ -0,0 +1 @@
+Boosts items in search result order by attributing higher scores for fields matching words or “tokens” used in query. Results are sorted by scores in descending order.
```

Comment:
Could we put the title of the document in here?

```suggestion
# Boosting

Boosts items in search result order by attributing higher scores for fields matching words or “tokens” used in query. Results are sorted by scores in descending order.
```

Reply:
We could also link through to the elastic documentation?
https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-boost.html